style.css

```diff
@@ -1,10 +1,5 @@
-/* stylelint-disable csstree/validator */
-
-
 /* stylelint-disable indentation */
 
```

Comment:
- [ ] Please don't disable the linters using inline comments instead I would highly recommend fixing the linter errors manually. Linters can help enforce a specific coding style, so code reviews are about the implemented changes and less about style or code formatting conventions. Moreover the use of linters helps to diagnose and fix technical issues—e.g. code smells—in the code. As a result, fewer defects make their way to production. Linters can help teams achieve a more readable and consistent style, through the enforcement of its rules.

Reply:
its the Vscode problem try to check properly unless if the linters did not pass im having problem with Vscode tried to do research about it nothing coming

Reply:
its not the Code problem its the  computer try to double check properly

Reply:
maybe its in your code cause the top part of the code its not there

Reply:
there is only one which is Vscode problem which is indentation